pipelines/pulpcore/02-install.yml

```
@@ -14,7 +14,7 @@
       state: present
     when:
      - pipeline_version is defined
-     - pipeline_version == 'nightly' or pipeline_version is version('3.28', '>=')
+     - pipeline_version == 'nightly' or pipeline_version is version('3.39', '>=')
```

This will break the 3.28 pipeline since it'll no longer install PG 12 there. 3.39 should match >= 3.28 anyway:
```suggestion
     - pipeline_version == 'nightly' or pipeline_version is version('3.28', '>=')
```